packages/@tinacms/fields/src/components/Toggle.tsx

```diff
@@ -29,34 +29,64 @@ export interface ToggleProps {
   input: any
   checked?: boolean
   disabled?: boolean
+  toggleLabels?:
+    | boolean
+    | {
+        true: string
+        false: string
+      }
 }
 
 export const Toggle: FC<ToggleProps> = props => {
   const checked = !!(props.input.value || props.input.checked)
+  const labels = {
+    true: props.toggleLabels === true ? 'Yes' : null,
+    false: props.toggleLabels === true ? 'No' : null,
+  }
+
+  if (typeof props.toggleLabels === 'object' && 'true' in props.toggleLabels) {
+    labels.true = props.toggleLabels['true']
+    labels.false = props.toggleLabels['false']
+  }
+
+  const hasToggleLabels =
+    typeof labels.true !== null || typeof labels.false !== null
```

Comment:
typeof `null` is always 'object'. We should adjust this to check that the labels equal typeof 'string'

Reply:
good catch. I'll fix it today.

Reply:
all good, I can update this as I address some other issues.

Reply:
sure!